app/javascript/components/pf_aggregate_status_card.jsx

```diff
@@ -31,7 +31,7 @@ const PfAggregateStatusCard = ({
       </h2>
       <div className="card-pf-body">
         <div className="card-pf-aggregate-status-notifications">
-          { (data.notifications && data.notifications[0].dataAvailable === false) ? (
+          { (data.notifications[0] && data.notifications[0].dataAvailable === false) ? (
```

Comment:
not sure if we should do like this
`data.notifications && data.notifications[0] && data.notifications[0].dataAvailable` 

for case like `data.notofications = []` and `data.notofications = null`

Reply:
@jeffibm fixed it